### ISSUE TITLE
On mobile, allow users to send a transaction by tapping on the transaction grid

### ISSUE THREAD

**Issue by rmshea** [solana-labs/break#141]
For iPhone X users, the "Send Transaction" button is rendered off screen. Is it possible to have users send a transaction by tapping anywhere on the screen during the 15 seconds (or just on the transaction grid)?

**Comment by jstarry** [comment]
Yeah, but if you accidentally tap a block, it will open the modal 🤔
